ql/src/test/results/clientpositive/load_dyn_part11.q.out

```diff
@@ -65,2012 +65,2012 @@ POSTHOOK: Input: default@nzhang_part_n0
 POSTHOOK: Input: default@nzhang_part_n0@ds=2010-03-03/hr=11
 #### A masked pattern was here ####
 238	val_238	2010-03-03	11
-86	val_86	2010-03-03	11
-311	val_311	2010-03-03	11
-27	val_27	2010-03-03	11
-165	val_165	2010-03-03	11
-409	val_409	2010-03-03	11
-255	val_255	2010-03-03	11
-278	val_278	2010-03-03	11
-98	val_98	2010-03-03	11
-484	val_484	2010-03-03	11
-265	val_265	2010-03-03	11
-193	val_193	2010-03-03	11
-401	val_401	2010-03-03	11
-150	val_150	2010-03-03	11
-273	val_273	2010-03-03	11
-224	val_224	2010-03-03	11
-369	val_369	2010-03-03	11
-66	val_66	2010-03-03	11
-128	val_128	2010-03-03	11
-213	val_213	2010-03-03	11
-146	val_146	2010-03-03	11
-406	val_406	2010-03-03	11
-429	val_429	2010-03-03	11
-374	val_374	2010-03-03	11
+97	val_97	2010-03-03	11
+200	val_200	2010-03-03	11
+400	val_400	2010-03-03	11
+403	val_403	2010-03-03	11
+169	val_169	2010-03-03	11
```

Comment:
I am assuming the results are same but in different order now?

Reply:
I manually checked and that is the conclusion I arrived at.